Formula/v8.rb

```diff
@@ -3,8 +3,10 @@
 class V8 < Formula
   desc "Google's JavaScript engine"
   homepage "https://github.com/v8/v8/wiki"
-  url "https://github.com/v8/v8-git-mirror/archive/5.1.281.47.tar.gz"
-  sha256 "63c9933227d6912689ea6bc012eea6a1fabaf526ac04bc245d9381e3ea238bf6"
+  version "6.5.254"
+
+  # NOTE: Hack because we need depot_tools and don't want to pull v8 twice.
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
I think it can actually work. What do you think about this (mis)use of the gclient cache?

https://github.com/chrmoritz/homebrew-core/commit/d82b9e0c30b781a2d87ec925ddfb37817e9d05c8

Reply:
I think it could work. I'll give it a try...